pycbc/inject/inject.py

```diff
@@ -624,8 +627,10 @@ def make_strain_from_inj_object(self, inj, delta_t, detector_name,
             f_l = f_lower
 
         # compute the waveform time series
+        logging.info('before waveform')
```

Comment:
Was this some debugging code that was left in, or should this be on the main branch?

Reply:
removed